modules/middleware/src/buildAggregations/createFieldAggregation.js

```diff
@@ -20,10 +24,57 @@ const createNumericAggregation = ({ type, field, graphqlField }) => {
   };
 };
 
-const createTermAggregation = ({ field, isNested }) => {
+const createTermAggregation = ({ field, isNested, graphqlField }) => {
+  const termFilter = graphqlField?.buckets?.filter_by_term || null;
+  const topHits = graphqlField?.buckets?.top_hits || null;
+  const source = topHits?.__arguments[0]?._source || null;
+  const size = topHits?.__arguments[1]?.size || 1;
```

Comment:
Probably should declare the Default value for top hits size above the same way you did for cardinality default.